src/module/actor/data-model-classes/data-model-character-encumbrance-item-based.ts

```diff
@@ -61,10 +61,16 @@ export default class OseDataModelCharacterEncumbranceItemBased
 
   #packedWeight;
 
+  #weightMod;
+
   // eslint-disable-next-line sonarjs/cognitive-complexity, @typescript-eslint/no-unused-vars
-  constructor(max = 16, items: Item[] = []) {
+  constructor(max = 16, items: Item[] = [], _ = "", strMod = 0) {
```

Comment:
Shouldn't you be able to use `_.scores.str.mod` instead of adding another argument?

Reply:
Thanks!